api/model/organization/UserHistoryPerOrganization.v1.yaml

```diff
@@ -0,0 +1,32 @@
+title: User history in organization
```

Comment:
L'intera struttura mi sembra un po' troppo innestata, ma non saprei come migliorarla...

Reply:
l'ideale sarebbe stato avere gli id come key, ma json non accetta integer :( e a sto punto tanto valeva essere espliciti. 

il bonus di strutturarlo così è che riutilizziamo quella interna in due posti (lato user e lato organization)